src/applications/ivc-champva/10-10D/config/submitTransformer.js

```diff
@@ -29,102 +31,83 @@ function transformRelationship(obj) {
   return rel;
 }
 
+// For each applicant, adjust organization of the object, add supporting documents array
 function transformApplicants(applicants) {
   const applicantsPostTransform = [];
-
   applicants.forEach(app => {
-    const transformedApp = {
-      fullName: app.applicantName ?? '',
+    let transformedApp = {
+      ...app,
       ssnOrTin: app.applicantSSN?.ssn ?? '',
-      dateOfBirth: fmtDate(app.applicantDob) ?? '',
-      phoneNumber: app.applicantPhone ?? '',
-      email: app.applicantEmailAddress ?? '',
```

Comment:
Really no reason to be changing the keynames for these (and other) properties, so revert to just using `...app` to include all the possible original properties. This requires updating the JSON mapping in the backend, which has been done in [this PR](https://github.com/department-of-veterans-affairs/vets-api/pull/18045).